### PR TITLE
Steering scripts for O2Os

### PR DESCRIPTION
**Pull request by ggovi**
Added scripts for o2o steering in offline and online environments.
Includes the version currently deployed.

### CODE REVIEW

**Comment by ggovi** [comment]
please test

**Comment by fabiocos** [comment]
@ggovi if this is needed in operation it should be signed and integrated asap. See my comment to the version for master about the location of the o2o directory (not a major issue in my view, just a suggestion)

File: CondCore/Utilities/o2o/deploy_offline.sh [hunk 1]
```diff
@@ -0,0 +1,79 @@
+#!/bin/sh                                                                                                                                                                 
+
+home=~
+localhome=/data/condbpro
+root=/data/O2O
+cmsswroot=/cvmfs/cms.cern.ch/
+extroot=/data/ext
+release=CMSSW_10_0_5
```
Comment (fabiocos) [inline CondCore/Utilities/o2o/deploy_offline.sh]: @ggovi sorry, is this intended for a 10_1_X release?

File: CondCore/Utilities/o2o/deploy_online.sh [hunk 2]
```diff
@@ -0,0 +1,89 @@
+#!/bin/sh                                                                                                                                                                 
+
+home=~
+localhome=/data/popconpro
+root=/data/O2O
+cmsswroot=/opt/offline
+extroot=/data/ext
+release=CMSSW_10_0_5
```
Comment (fabiocos) [inline CondCore/Utilities/o2o/deploy_online.sh]: @ggovi same here...

File: CondCore/Utilities/o2o/deployed/offline/setupO2O.sh [hunk 3]
```diff
@@ -0,0 +1,100 @@
+#!/bin/sh
+
+# deployment-specific params
+BASEDIR=/data/O2O
+RELEASE=CMSSW_10_0_5
```
Comment (fabiocos) [inline CondCore/Utilities/o2o/deployed/offline/setupO2O.sh]: @ggovi and here...

File: CondCore/Utilities/o2o/deployed/online/setupO2O.sh [hunk 4]
```diff
@@ -0,0 +1,100 @@
+#!/bin/sh
+
+# deployment-specific params
+BASEDIR=/data/O2O
+RELEASE=CMSSW_10_0_5
```
Comment (fabiocos) [inline CondCore/Utilities/o2o/deployed/online/setupO2O.sh]: @ggovi here as well

File: CondCore/Utilities/o2o/deploy_offline.sh (hunk 1 above)
Comment (ggovi) [reply CondCore/Utilities/o2o/deploy_offline.sh]: Well it is intended for any release, depending on the deployment needs. All of the parameters of the deploy are placeholders here, and they should be properly set when the script is executed.

File: CondCore/Utilities/o2o/deploy_online.sh (hunk 2 above)
Comment (ggovi) [reply CondCore/Utilities/o2o/deploy_online.sh]: same as for deploy_offline

File: CondCore/Utilities/o2o/deployed/offline/setupO2O.sh (hunk 3 above)
Comment (ggovi) [reply CondCore/Utilities/o2o/deployed/offline/setupO2O.sh]: these are examples of the scripts created by deploy_online, currently running at P5

File: CondCore/Utilities/o2o/deployed/online/setupO2O.sh (hunk 4 above)
Comment (ggovi) [reply CondCore/Utilities/o2o/deployed/online/setupO2O.sh]: same as before

**Comment by fabiocos** [comment]
@ggovi could you please sign this PR? I understand it is more for reference than for use out of the box, but anyway it does not create issues.

**Comment by fabiocos** [comment]
+1

fully signed in 10_2_X, not affecting standard workflow, useful for reference in operations

**Pull request merged**